saleor/csv/utils/products_data.py

```diff
@@ -403,8 +413,13 @@ def add_attribute_info_to_data(
             value = f"{attribute_data.value}"
             if attribute_data.unit:
                 value += f" {attribute_data.unit}"
-        elif input_type == AttributeInputType.RICH_TEXT:
+        elif input_type == AttributeInputType.RICH_TEXT and attribute_data.rich_text:
```

Comment:
Why you added `attribute_data.rich_text` to the condition? It's not needed. When the value is `None` the `clean_editor_js` will return empty string. We don't want to get value for this attribute from `clean_editor_js`.